torch/utils/_config_module.py

```diff
@@ -432,19 +454,20 @@ def _get_dict(
             skip_default does two things. One if a key has not been modified
                 it skips it. The other is it modified the logging behaviour
                 to match what codegen already did for modified skipped keys
+            env_default: If set, configs overriden by the env are not considered default.
```

Comment:
I think this should always be True?

Reply:
Currently, the test is written such that these configs are skipped by skip_default. If we change this to always be True, we need to change the existing tests. I'm checking with test writer (Colin) if we can change it.

Reply:
I'll change it to always be True